contrib/buildrefactor/src/python/pants/contrib/buildrefactor/peek.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+#./pants peek --path=src/scala/org/pantsbuild/zinc/analysis --target-name=zinc-analysis
+#./pants peek --path=src/scala/org/pantsbuild/zinc/extractor --target-name=zinc-extractor --line_number=True
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.task.task import Task
+from pants.contrib.buildrefactor.buildozer import Buildozer
+
+
+class Peek(Task):
+    """Peek a build file for a specified target
```

Comment:
Can you write down here whether the functionality here tells you the literal value in the BUILD file, or tells you the value that pants will actually use, or something else? Each of these options are potentially useful, but noting which one we expect is handy.

For instance, in the following:

```
dir$ cat BUILD
java_library(
  dependencies = [
    "//foo:bar",
    "//baz:bash",
  ],
)

dir$ ls -1
BUILD
Foo.java
```

there are a bunch of things which Pants will internally do which aren't seen in the BUILD file. Some examples:

1. The `name` attribute will be populated with the name of the directory containing the BUILD file. Say the directory is named "dir"; should peek return "dir", "", some special value which denote "missing", some special value which denotes "computed", or an error?
2. The `srcs` attribute will automatically be expanded to `globs("*.java")`. Should peek error / return "" (because it's absent), return `globs("*.java")` because that's what's in the BUILD file, or call the function `globs` and return the list of java files which it actually matches (i.e. `["Foo.java"]`)?
3. Ordering isn't significant in the dependencies attribute, and Pants internally will sort the `dependencies` attribute. Should we return the unsorted list (because that's what in the BUILD file), or a sorted version (because that's what Pants will actually use, and it makes comparing two targets easier)?

@wisechengyi I don't know what you're expecting the answers to these questions to be either, or whether you've had this discussion already :)

Reply:
The current behavior is to print the literal content in BUILD file and not what Pants sees, so just
```
java_library(
  dependencies = [
    "//foo:bar",
    "//baz:bash",
  ],
)
```
which is fine I think.

Reply:
Totally fine, but worth writing down in the help text :)